web/src/scripts/webflow/industrySync.mjs

```diff
@@ -0,0 +1,171 @@
+  );
+};
+
+const convertIndustryToWebflowIndustryName = (industry) => {
+  return {
+    name: industry.name,
+    slug: industry.id,
+    additionalsearchterms: industry.additionalSearchTerms,
+    description: industry.description,
+    industryquerystring: industry.id,
+  };
+};
+
+const updateWebflowIndustryNames = async (industries) => {
+  const modify = async (industry) => {
+    console.info(`Attempting to modify ${industry.name}`);
+    try {
+      const webflowIndustryName = convertIndustryToWebflowIndustryName(industry);
+      return await modifyItem(industry.webflowId, industryNameCollectionId, webflowIndustryName);
+    } catch (error) {
+      console.error(error.response);
+      throw error;
+    }
+  };
+
+  await Promise.all(
+    industries.map(async (industry) => {
+      return await modify(industry);
+    })
+  );
+
+  console.info(`Modified a total of ${industries.length} Webflow Industry Names`);
+};
+
+const loadNavigatorIndustry = (fileName) => {
+  const fullPath = path.join(navigatorIndustryDir, `${fileName}`);
+  const content = fs.readFileSync(fullPath, "utf8");
+  const industry = JSON.parse(content);
+  return industry;
+};
+
+const updateIndustryWithWebflowId = (webflowId, filename) => {
+  const industry = loadNavigatorIndustry(`${filename}.json`);
+  const updatedIndustry = {
+    ...industry,
+    webflowId,
+  };
+  const jsonString = JSON.stringify(updatedIndustry, null, 2);
+
+  fs.writeFileSync(`${navigatorIndustryDir}/${filename}.json`, jsonString, (err) => {
+    if (err) {
+      throw err;
+    }
+  });
+};
+
+const createNewWebflowIndustryNames = async () => {
+  const newIndustries = await getNewIndustries();
+
+  const create = async (industry) => {
+    console.info(`Attempting to create ${industry.name}`);
+    let result;
+    try {
+      const webflowIndustryName = convertIndustryToWebflowIndustryName(industry);
+      result = await createItem(webflowIndustryName, industryNameCollectionId, false);
+    } catch (error) {
+      console.error(error.response);
+      throw error;
+    }
+
+    if (industry.webflowId === undefined) {
+      const filename = industry.id;
+      const webflowId = result.data._id;
+      updateIndustryWithWebflowId(webflowId, filename);
+    }
+    return Promise.resolve();
+  };
+
+  await Promise.all(
+    newIndustries.map(async (industry) => {
+      return await create(industry);
+    })
+  );
+
+  console.info(`Created a total of ${newIndustries.length} Webflow Industry Names`);
+};
+
+const syncIndustries = async (params) => {
+  console.log("updating industry names");
+  const industriesAlreadyInWebflow = await getIndustryNamesAlreadyInWebflow();
+
+  await updateWebflowIndustryNames(industriesAlreadyInWebflow);
+  console.log("creating new industry names");
+
+  if (params.create) {
+    await createNewWebflowIndustryNames();
+    console.log("Complete industry sync!");
```

Comment:
[dust]: You're using console.logs in some areas and console.info in others. Feels inconsistent to me? But perhaps there is a reason I'm missing?

Reply:
There's no reason. There's some copy/paste going on here. I'll make these all `console.log()`. I can see where someone might argue one or the other, but I don't think it matters for our purposes.